modules/administration/pages/kubernetes.adoc

```diff
@@ -1,175 +1,173 @@
+* ``Outdated instances found``:
++
+Some of the instances are running an older build of the image.
+You might need to redeploy the image into the pod.
+* ``No information``:
++
+The checksum of the instance image does not match the image data contained in {productname}.
+You might need to redeploy the image into the pod.
 
 
-The following steps will help you build an image for deployment in Kubernetes.
 
-. Under menu:Images[Stores], create an image store.
-. In menu:Images[Profiles], create an image profile (with a dockerfile that is suitable to deploy to Kubernetes).
-. Under menu:Images[Build], build an image with the new profile and wait for the build to finish.
-. Deploy the image into one of the registered Kubernetes clusters using ``kubectl``.
+// This procedure needs help. LKB 2019-10-03
+.Procedure: Building an Image
+. In the {productname} {webui}, navigate to menu:Images[Stores].
+. Click btn:[Create] to create an image store.
+. Navigate to menu:Images[Profiles].
+. Click btn:[Create] to create an image profile.
+You will need to use a dockerfile that is suitable to deploy to {k8s}.
+. Navigate to menu:Images[Build] to build an image with the new profile.
+. Use the [command]``kubectl`` tool to deploy the image into one of the registered {k8s} clusters.
 
-In the `Runtime` and `Instances` columns in the respective image row you can now see the updated data.
+The updated data should now be available in the image list at menu:Images[Image List].
 
 
 
-[[at.import.a.previously.deployed.image.in.kubernetes]]
-== Import a Previously Deployed Image in Kubernetes
+// This procedure needs help. LKB 2019-10-03
+.Procedure: Importing a Previously Deployed Image
+. In the {productname} {webui}, navigate to menu:Images[Image List], and select the existing image that you want to import.
+. Add the registry owning the image as an image store.
+. Navigate to menu:Images[Image List] and click btn:[Import].
+. Complete the fields and click btn:[Import].
 
+The updated data should now be available in the image list at menu:Images[Image List].
 
-The following steps will guide you through importing a previously deployed image in Kubernetes.
 
-. Select an image that has already been deployed to any of your registered Kubernetes clusters.
-. Add the registry owning the image to SUSE Manager as an image store.
-. Navigate to menu:Images[Image List], click `Import` from the top-right corner, fill in the form fields and click ``Import``.
 
-In the `Runtime` and `Instances` columns in the respective image row you can now see the updated data.
+.Procedure: Rebuilding a Previously Deployed Image
 
+. In the {productname} {webui}, navigate to menu:Images[Image List], locate the row that contains the instance you want to rebuild, and click btn:[Details].
+. Navigate to the [guimenu]``Build Status`` section, and click btn:[Rebuild].
 
+The runtime status of the image should now be updated in the image list at menu:Images[Image List].
```

Comment:
This happens after the build is successfully finished, which may take minutes after the user clicks the Rebuild button.